exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,7 @@
-const name = " Daniel  ";
+var name = "  My Name is Lawrence!   ";
+var nameLength = name.length;
+
+console.log(name.trim(), nameLength);
```

Comment:
Hi Lawrence, the expected result for this is "My name is Daniel and my name is 6 characters long"

Reply:
Hi Samuel. i have corrected the expected result and changed "Lawrence" to "Daniel".